.gitignore

```diff
@@ -12,7 +12,6 @@ built-docs/
 .DS_Store
 .vstags
 
-package-lock.json
```

Comment:
I thought this was ignored already. I wonder how it got back 🤔

Reply:
As discussed with @azangru offline, we need to this file for continuous integration (CI).